certified-connectors/FORCAM FORCE Bridge/apiDefinition.swagger.json

```diff
@@ -0,0 +1,5073 @@
+        "tags": [
+          "ticket"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "operationId": "TicketCreate",
+        "parameters": [
+          {
+            "name": "newTicket",
+            "in": "body",
+            "required": true,
+            "description": "Ticket to create",
+            "schema": {
+              "type": "object",
+              "required": [
+                "priority",
+                "ticketStateId",
+                "title"
+              ],
+              "properties": {
+                "title": {
+                  "type": "string"
+                },
+                "description": {
+                  "type": "string"
+                },
+                "rootCause": {
+                  "type": "string"
+                },
+                "priority": {
+                  "type": "string",
+                  "enum": [
+                    "LOW",
+                    "MEDIUM",
+                    "HIGH",
+                    "VERY_HIGH"
+                  ]
+                },
+                "ticketStateId": {
+                  "type": "string",
+                  "enum": [
+                    "NEW",
+                    "IN_PROGRESS",
+                    "DONE",
+                    "CONFIRMED",
+                    "REOPENED"
+                  ]
+                },
+                "ticketClassId": {
+                  "type": "string",
+                  "x-ms-dynamic-values": {
+                    "operationId": "TicketClasses",
+                    "value-path": "id",
+                    "value-title": "description",
+                    "value-collection": "ticketClasses"
+                  }
+                },
+                "operationId": {
+                  "type": "string"
+                },
+                "materialNumber": {
+                  "type": "string"
+                },
+                "workplaceId": {
+                  "type": "string",
+                  "x-ms-summary": "WorkplaceId",
+                  "x-ms-dynamic-values": {
+                    "operationId": "getWorkplaces",
+                    "value-path": "id",
+                    "value-title": "name",
+                    "value-collection": "workplaces"
+                  }
+                },
+                "shiftId": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Ticket"
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/tickets/ticketId/{ticketId}": {
+      "get": {
+        "description": "Get ticket by Id",
+        "summary": "Get ticket by Id ",
```

Comment:
@dominikamann Please remove extra space at the end of the string

Reply:
@sriyen-msft Resolved, thank you for the information.